benchmarks/src/has_single_bit.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <benchmark/benchmark.h>
+#include <bit>
+
+#include <utility.hpp>
+
+template <typename T>
+static void has_single_bit_if(benchmark::State& state) {

```

Comment:
`static` linkage were dropped in existing benchmarks in #5370 along with unnamed namespace, because they serve no purpose for a single TU program. Let's not introduce them here. Also occurs once below.

Reply:
done.